src/views/partials/gallery.ejs

```diff
@@ -0,0 +1,22 @@
+<section class="gallery-wrapper">
+    <ul class="slider">
+        <li class="item-slide">
+            <img class="gallery-highlight" src=<%= prd.product_image %> alt="main image">
```

Comment:
as per the previous suggestion please update to: 
```suggestion
            <img class="gallery-highlight" src=<%= product.product_image %> alt="main image">
```

Reply:
Done